### ISSUE TITLE
"installdir" field should be set

### ISSUE THREAD

**Issue by MingweiSamuel** [dotfloat/steam-appmanifest#15]
I installed NEKOPARA Vol. 0 using the python script. However, when I looked at my `common` folder, I saw that all the files were thrown there instead of being in a separate folder, like all the other games. Looking at the appmanifest files, I saw that the `"installdir"` field was set to `""`, instead of something useful. 

The script should supply a `"installdir"` field if it is blank to prevent this from happening.

**Comment by dotfloat** [comment]
Does it work when you set the `installdir` manually before starting Steam?

**Comment by MingweiSamuel** [comment]
I will test that tomorrow

**Comment by AbigailBuccaneer** [comment]
https://github.com/leovp/steamfiles can parse `~/.steam/steam/appcache/appinfo.vdf` to find the canonically correct installdir.

**Comment by dotfloat** [comment]
Oh wow, this is great. Time to rewrite this project.

Am I correct in thinking that the `appinfo.vdf` file contains all of the user's programs, regardless if they're installed or not?

**Comment by AbigailBuccaneer** [comment]
It seems to contain all of the user's games and tools, regardless of whether they're installed or whether they're available for the current platform.

**Comment by dotfloat** [comment]
I have started a separate project over at https://github.com/dotfloat/Vapour, which is intended to be a serious thing, unlike this `appmanifest` hack.

Basically what I want is to be able to write `vapour install --windows "ultimate doom"` and have it use SteamCMD to download it somewhere without restarting Steam. (And eventually have a GUI, of course.)

I could also do auto-updates and launch games using source ports or wine. ;)

It'll take a few weeks before it's ready, but I think I'll leave this issue as WONTFIX, unless someone else wants to have fun fixing it.

**Comment by directhex** [comment]
> unless someone else wants to have fun fixing it.

https://github.com/dotfloat/steam-appmanifest/pull/17
